src/JuMP.jl

```diff
@@ -395,6 +395,30 @@ function add_bridge(model::Model,
     return
 end
 
+"""
+    empty!(model::Model) -> model
+
+Empty the model, that is, remove all variables, constraints and model
+attributes but not optimizer attributes. Always return the argument.
+
+Note: removes extensions data.
+"""
+function Base.empty!(model::Model)::Model
+    # The method changes the Model object to, basically, the state it was when
+    # created (if the optimizer was already pre-configured). The two exceptions
+    # are: optimize_hook and operator_counter. One is left alone because it is
+    # related to optimizer attributes and the other is just a counter for a
+    # single-time warning message (so keeping it helps to discover
+    # inneficiencies).
+    MOI.empty!(model.moi_backend)
+    empty!(model.shapes)
+    empty!(model.bridge_types)
```

Comment:
I would keep `brodge_types` unaltered to be consistent witn `MOI.empty!` for `MOI.Bridges.LazyBridgeOptimizer`

Reply:
Done. Commit message is missing a `model.bridge_types` after the comma because bash interpreted it as a subshell invocation (sorry) and I did not perceive until after push.

Reply:
I tested with just the tests in `model.jl`, those passed.